src/include/distributed/commands/utility_hook.h

```diff
@@ -30,6 +30,7 @@ typedef enum
 } PropSetCmdBehavior;
 extern PropSetCmdBehavior PropagateSetCommands;
 extern bool EnableDDLPropagation;
+extern bool AllCitusLocalTable;
```

Comment:
What is this new variable? We seem to not use it?

Reply:
this was part of vanilla test, removing it :)